src/lib/MovingAverage.java

```diff
@@ -0,0 +1,40 @@
+package lib;
+
+import java.util.LinkedList;
+public class MovingAverage {
+	private static LinkedList<Double> queue;
+	private static int windowSize;
+	private double curTotal;
+	
+	/** 
+	 * Creates the average 
+	 * @param size  used to create the average   
+	 */
+	public MovingAverage(int windowSize){
+		if (windowSize <= 0){
+			throw new IllegalArgumentException(windowSize + "is not a valid");
```

Comment:
This error message could read "-1 is not a valid".  This error should be readable.  Please fix